### DISCUSSION TITLE
【launch failed】 Upgrade from version 3.3.2 to version 3.6.0, startup failed. How to implement old version devices, it is important not to lose them

### DISCUSSION THREAD

**Discussion by hhhaiai** [Genymobile/feedback#89] [Genymotion Desktop] [answered]
Upgrade from version 3.3.2 to version 3.6.0, startup failed. How to implement old version devices, it is important not to lose them

[genymotion-logs-20240410-113837.zip](https://github.com/Genymobile/feedback/files/14926740/genymotion-logs-20240410-113837.zip)

**Comment by jmaneyrol69** [comment, marked as answer]
Hi @hhhaiai,

We are sorry but this board is strictly reserved to feature requests and suggestions. It is not suitable for technical assistance.
Please visit our help center at https://support.genymotion.com/hc/en-us/

Thank you.
